configs/experiments/relative_lr/small_relative_lr.yaml

```diff
@@ -0,0 +1,28 @@
+parent: configs/baselines/gpt/dense/medium.yaml
+md5_parent_hash: 1706dd43c527be74a6282b0319e9c31e
+time: "9:00:00"
+
+params:
+  # tokenizer: gpt
+
+  name: "relative_lr_medium"
+  grad_clip: 1.0
+  flash_attention: false
+  mixed_precision: false
+
+  lr_ratio_modules: ["embedding_layer", "head"]
+  lr_ratios: [5, 0.2]
+  tags: ["relative_lr", "check_tuned_lr_with_relative"]
```

Comment:
Can this look like:
```
lr_ratios:
          embedding_layer: 5
          head: 0.2
```
This will allow us to grid normally

Reply:
Done